RegularApi.Tests/Dao/BaseDaoIT.cs

```diff
@@ -1,29 +1,38 @@
+using Microsoft.Extensions.DependencyInjection;
 using MongoDB.Driver;
+using RegularApi.Tests.Fixtures;
 
 namespace RegularApi.Tests.Dao
 {
     public abstract class BaseDaoIT : BaseIT
     {
-        private static string _mongoDatabaseName = "regularOrchestrator";
+        private const string MongoDatabaseName = "regularOrchestrator";
 
-        protected void DropCollection(string name)
-        {
-            GetDatabase().DropCollection(name);
-        }
+        protected void DropCollection(string name)
+        {
+            GetDatabase().DropCollection(name);
+        }
 
         protected IMongoCollection<T> GetCollection<T>(string name)
         {
             return GetDatabase().GetCollection<T>(name);
-        }
+        }
 
         protected T GetDao<T>()
         {
-            return (T)ServiceProvider.GetService(typeof(T));
-        }
-
-        private IMongoDatabase GetDatabase()
-        {
-            return MongoClient.GetDatabase(_mongoDatabaseName);
+            return ServiceProvider.GetRequiredService<T>();
+        }
+
+        protected DaoFixture GetDaoFixture()
+        {
+            return GetDao<DaoFixture>();
+        }
+
+        // ------------------------------------------------------------------------------------
```

Comment:
Could you remove this line? in case that it's necessary do a separation here, I think the method below not belong here

Reply:
It is used to make a visual separation on different context into the code